lib/google-contacts/element.rb

```diff
@@ -289,7 +309,10 @@ def update_groups(*group_links)
     end
 
     private
-     # Evil ahead
+      def get_google_type(google_type)
```

Comment:
@rohandaxini sure I'll change method name to `get_google_label_name`.

`# Evil ahead` this comment was there previously. I have removed it in my changes. :stuck_out_tongue_winking_eye: